app/modules/users/user_router.py

```
@@ -20,16 +20,19 @@ class UserAPI:
     @router.get(
         "/user/conversations/",
         response_model=List[UserConversationListResponse],
+        description="Get a list of conversations for the current user with sorting options."
     )
     async def get_conversations_for_user(
         user=Depends(AuthService.check_auth),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)